src/main/java/seedu/commands/RemoveCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.commands;
+
+import seedu.data.ingredient.Ingredient;
+import seedu.parser.Parser;
+import seedu.storage.IngredientList;
+import seedu.ui.Ui;
+
+import java.util.ArrayList;
+
+public class RemoveCommand extends Command{
```

Comment:
Got error here, space between Command and  {

Reply:
Resolved.